### DISCUSSION TITLE
Code tutorial: Transcribe pre-recorded audio for python does not work

### DISCUSSION THREAD

**Discussion by elhoim** [deepgram/community#511] [General help] [answered]
### Which Deepgram product are you using?

Python SDK

### Details

So from what i can see is that the class Deepgram has been deprecated in v3 of the SDK and instead the DeepgramClient should be used. I had to read the code to realize this, since instead of getting the Exception raised that was written in it, i had the following error in python:
`Traceback (most recent call last):
  File ".\1032.DeepGram-Playground\main.py", line 24, in <module>
    main()
  File ".\1032.DeepGram-Playground\main.py", line 12, in main
    dg_client = Deepgram(DEEPGRAM_API_KEY)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Deepgram.__init__() takes 1 positional argument but 2 were given
`

The way of passing options has also completely changed and requiring a DeepgramClientOptions class now.

### If possible, please attach your code or paste it into the text box.

Code is coming from the mission: https://console.deepgram.com/project/61365818-76c1-4a2e-88c4-fffb46d8491d/mission/transcript-via-api-or-sdk

```
from deepgram import Deepgram
import json

# The API key we created in step 3
DEEPGRAM_API_KEY = '[CENSORED]'

# Hosted sample file
AUDIO_URL = "https://static.deepgram.com/examples/L_intelligence_artificielle.mp3"

def main():
    # Initializes the Deepgram SDK
    dg_client = Deepgram(DEEPGRAM_API_KEY)
    source = {'url': AUDIO_URL}
    options = { "punctuate": True, "model": "nova-2", "language": "fr" }

    print('Requesting transcript...')
    print('Your file may take up to a couple minutes to process.')
    print('While you wait, did you know that Deepgram accepts over 40 audio file formats? Even MP4s.')
    print('To learn more about customizing your transcripts check out developers.deepgram.com')

    response = dg_client.transcription.sync_prerecorded(source, options)
    print(json.dumps(response, indent=4))

main()

```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment, marked as answer]
The Playground/Console updates are still pending. There is an issue filed here:
https://github.com/deepgram/deepgram-python-sdk/issues/237

Most of of the other documentation has been updated already. Other users pointed out areas I had missed but have since been addressed:
https://github.com/deepgram/deepgram-python-sdk/issues/236

**Comment by davidvonthenen** [comment]
The console and playground code should be updated now.
